### PR TITLE
User can view disc inventory

### PR DESCRIPTION
**Pull request by marshall-vogt**
Screenshot 2023-08-17 at 10 08 42 AM
Screenshot 2023-08-17 at 10 08 22 AM
Screenshot 2023-08-17 at 10 08 53 AM

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+.header-title {
+  width: 100%;
+}
+
+.home-page {
+  position: fixed;
+  bottom: 0;
+  z-index: -1;
+}
```
Comment (shawnkost) [inline css/layout.css]: Your `home-page` shouldn't be `position: fixed`. That would remove it from the document flow. Only certain things should be used alongside positioning, like a modal, popup, etc..

File: css/layout.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+.home-page-button {
+  position: fixed;
+  bottom: 5%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
```
Comment (shawnkost) [inline css/layout.css]: I don't think this should be fixed either

File: css/layout.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+.discs {
+  padding-top: 110px;
+}
```
Comment (shawnkost) [inline css/layout.css]: You should be avoiding using such high `padding` values. This usually means you're doing something "hacky" to get this to work.

**Comment by marshall-vogt** [comment]
Screenshot 2023-08-17 at 10 21 03 AM
Screenshot 2023-08-17 at 10 21 13 AM
Screenshot 2023-08-17 at 10 21 24 AM

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+  $homePage.classList.add('hidden');
+  $searchPage.classList.remove('hidden');
+  $searchPage.classList.add('shown');
+}
+
+const $navbarButtons = document.querySelectorAll('i');
+const $homeButton = $navbarButtons[0];
+const $searchButton = $navbarButtons[1];
+// const $favoritesButton = $navbarButtons[2];
+// const $bagButton = $navbarButtons[3];
```
Comment (shawnkost) [inline js/main.js]: Delete old code

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+      $discName.textContent = sortedResponse[i].name;
+      $discNumbers.textContent = `${sortedResponse[i].speed}|${sortedResponse[i].glide}|${sortedResponse[i].turn}|${sortedResponse[i].fade}|`;
+      $discDiv.setAttribute('class', `disc ${sortedResponse[i].brand_slug}`);
+      $discName.setAttribute('class', 'disc-name');
+      $discNumbers.setAttribute('class', 'flight-numbers');
+      previousRow.append($discDiv);
+      $discDiv.append($discName);
+      $discDiv.append($discNumbers);
+    }
+  }
```
Comment (shawnkost) [inline js/main.js]: I would break this up into a separate function. Like what we do with `renderEntry` in Code Journal.

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+$searchPage.addEventListener('click', handleDiscClick);
+const $blurModal = document.querySelector('.background-blur-modal');
+$blurModal.addEventListener('click', removeBlur);
+const $discModal = document.querySelector('.disc-modal');
+const $flightPatternImage = document.querySelector('.flight-pattern-image');
+const $anchor = document.querySelector('a');
+
+async function handleDiscClick(event) {
+  const response = await fetch('https://discit-api.fly.dev/disc');
+  const data = await response.text();
```
Comment (shawnkost) [inline js/main.js]: Until you have a good understanding of promises and how they work, you should use `XMLHttpRequest` instead

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Marshall!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**